### ISSUE TITLE
CCNodeGradient bug in Yosemite

### ISSUE THREAD

**Issue by clickstan** [spritebuilder/SpriteBuilder#1020]
Found a bug in the latest Spritebuilder, after  updating to OS X Yosemite. When changing the start or end color on a CCNodeGradient, the other color changes to the default blue, no matter what color was selected.

**Comment by NickyWeber** [comment]
Weird, was able to reproduce this only once. Hitting save might have solved it. Can you reproduce this every time?

**Comment by clickstan** [comment]
I just tried, it won't happen again. Weird. The version of spray builder was the one in the app store, version 1.2.1. Saving and reopening the project seems to fix it.

**Comment by clickstan** [comment]
I tried reproducing it, and through my attempts, I got it to happen once, but after that it stopped happening. I will post if I figure out a way to reproduce it.

**Comment by NickyWeber** [comment]
Ok thanks, yeah same happened here. I got a feeling that as soon as the values for the gradient are saved to the model this is not happening any more.

**Comment by NoorFatima** [comment]
This issue is fixed.
Device: Mac OS X 10.9.5
SB Version: 1.4.7

**Comment by clickstan** [comment]
This issue happens only on mac os 10.10+, on 10.9 it doesn't happen. It's not fixed in SB 1.4.7, I checked today. I have another issue report for this issue, with reproduction instructions. Please check this: https://github.com/spritebuilder/SpriteBuilder/issues/1225
